pkgs/development/libraries/ffmpeg/generic.nix

```diff
@@ -470,6 +468,7 @@ stdenv.mkDerivation (finalAttrs: {
     (enableFeature withMysofa "libmysofa")
     (enableFeature withOpus "libopus")
     (optionalString (versionAtLeast version "5.0" && withLibplacebo) "--enable-libplacebo")
+    (optionalString (versionAtLeast version "6.1" && withLibaribcaption) "--enable-libaribcaption")
```

Comment:
I'd prefer using optionals here but I'll probably just clean this up myself.

Reply:
Can you elaborate?